modules/KalturaSupport/components/chapters/chapters.js

```diff
@@ -327,15 +332,16 @@
 			var itemsToRemoveIndexes = [];
 
 			var currentTime = this.getPlayer().getPlayerElementTime();
+			var _this = this;
```

Comment:
1.if (_this.getPlayer().isDvrSupported()) {
    currentTime = _this.getPlayer().LiveCurrentTime) 
}
2. why getPlayerElementTime() is not same as LiveCurrentTime . in lvie and DVR